.azure-pipelines/jobs/test-ubuntu.yml

```diff
@@ -24,11 +24,12 @@ jobs:
         displayName: "Run tests"
         env:
           TF_ACC: 1
-          CTP_AUTH_URL: http://localhost:8989
-          CTP_PROJECT_KEY: unittest
```

Comment:
Are the values `unittest` and `x` just random values or do they need to be like this?

Reply:
They need to be like this, it's running a docker container with this server: https://github.com/labd/commercetools-python-sdk/blob/master/src/commercetools/testing/server.py 

This way we can test without talking to an actual Commercetools instance.

Reply:
Ok, I changed the test values back to the original form.